### ISSUE TITLE
VA Forms System Core [Component]: Focus Management - No scroll back to the top of the content area when stepping through the form

### ISSUE THREAD

**Issue by TiffanyPender** [department-of-veterans-affairs/va-forms-system-core#546]
-[508-defect-1: ❗️Critical. Must be fixed before launch](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-508-defect-1)

## User Story or Problem Statement

As someone who navigates with a keyboard or mouse, when navigating a form that takes me through different steps and pages via BACK and CONTINUE style buttons, I expect for the screen to shift to the top of the form content of the page post-click/select. This helps me understand where on the page I am in relation to all content and keep my understanding of the flow of the form in tact.

## Details

When clicking on either the BACK or CONTINUE button on any step of the form, the screen does not shift focus to scroll up to the top of the form content on the loaded page.

## Acceptance Criteria

- [ ] The screen scrolls to the top of the form content after clicking the BACK or CONTINUE buttons on any form step.

## Environment

* macOS Monterey, `<Version 12.4>`
* Chrome, `<Version 102.0.5005.61>`
* VoiceOver
* Staging

## Steps to Recreate

1. Visit the [Forms Library Core demo form](https://staging.va.gov/burial-poc-v6/)
2. Navigate with either a keyboard or mouse to the CONTINUE button and click/select.
3. Verify that the page does not scroll to the top of the new form content.
    a. In Chrome Dev Tools, you can verify the active focus by using [live expression](https://developer.chrome.com/docs/devtools/accessibility/focus/) and tracking the active element with `document.activeElement`
4. Repeat step two on the remaining form pages, testing with clicking on both the BACK and CONTINUE buttons.

## WCAG or Vendor Guidance (optional)

* [When to manage focus in an accessible way](https://adhoc.team/2021/09/09/when-to-manage-focus-in-an-accessible-way/)

**Comment by obliviga** [comment]
@TiffanyPender Should focus be set on the `form` element itself, or on the `h1` element? Thanks!

**Comment by TiffanyPender** [comment]
@obliviga My recommendation is to either set focus on the `h1` element or on the step indicator. This will help screen reader users know that they have moved on to the next page of the form and will help other users to visually see the top/beginning of the new form content.

**Comment by obliviga** [comment]
@TiffanyPender, thanks. I'll set focus on the `h1` since the step indicator is a design system component that cannot be focused on. If an `h1` inside the form does not exist, then focus will be set on the default location, which is the `body`. Let me know what you think :)

**Comment by obliviga** [comment]
Verified that this fix has been deployed to the main branch.
